mne_bids/tests/test_mne_bids.py

```diff
@@ -51,10 +53,14 @@ def test_fif():
     events_fname = op.join(data_path, 'MEG', 'sample',
                            'sample_audvis_trunc_raw-eve.fif')
 
-    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
-                acquisition=acq, task=task, raw_file=raw_fname,
-                events_data=events_fname, output_path=output_path,
-                event_id=event_id, overwrite=True)
+    kwargs = dict(session_id=session_id, run=run,
+                  acquisition=acq, task=task, output_path=output_path,
+                  events_data=events_fname, event_id=event_id, overwrite=True)
+
+    raw_to_bids(subject_id=subject_id, raw_file=raw_fname, **kwargs)
```

Comment:
you should include an assertion on the type to make sure it's working properly for a given modality (e.g. 'fif' for `meg`, '.edf' for eeg